src/controllers/expenses.js

```diff
@@ -0,0 +1,90 @@
+'use strict';
+
+const expensesService = require('./services/expenses');
+
+const getAll = (req, res) => {
+  const expens = expensesService.getAll();
+
+  if (!expens) {
+    res.sendStatus(400);
+
+    return;
+  }
+
+  res.send(expens);
+};
+
+const getOne = (req, res) => {
+  const { userId } = req.params;
+
+  if (!userId) {
+    res.sendStatus(400);
+
+    return;
+  }
+
+  const foundExpens = expensesService.getById(userId);
+
+  if (!foundExpens) {
+    res.sendStatus(404);
+
+    return;
+  }
+
+  res.send(foundExpens);
+};
+
+const add = (req, res) => {
+  const { name } = req.body;
+
+  if (!name) {
+    req.sendStatus(422);
+
+    return;
+  }
+
+  const newExpens = expensesService.create(name);
+
+  res.statusCode(201);
```

Comment:
Use `res.status(201)` instead of `res.statusCode(201)` to correctly set the status code.